scripts/build/debug/300-gdb.sh

```diff
@@ -69,11 +69,8 @@ do_debug_gdb_build() {
 
         cross_extra_config=("${extra_config[@]}")
         cross_extra_config+=("--with-expat")
-        # NOTE: DO NOT USE --with-libexpat-prefix (until GDB configure is smarter)!!!
```

Comment:
> What is the change in behaviour between these two configurations?

Sorry, in prev comment I forgot to describe the main case:
If we do not pass "--without-expat-prefix" nor "--with-expat-prefix" flag, configure will look for expat in $prefix/lib (this is correct behavior), and in our case $prefix=/usr/lib, and voila - we got incorrect libexpat.so